emoji/src/main/java/com/vanniktech/emoji/EmojiManager.java

```diff
@@ -2,10 +2,13 @@
 
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
+
 import com.vanniktech.emoji.emoji.Emoji;
 import com.vanniktech.emoji.emoji.EmojiCategory;
 import com.vanniktech.emoji.emoji.EmojiTree;
 
+import java.util.List;
+
```

Comment:
`checkstyle` complains when removing the space between normal and static imports, so I will keep it as it is.